lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -32,17 +32,25 @@
 # The header in the request
 rokwire_api_key_header = 'rokwire-api-key'
 # Group names for the event and app config manager. These typically come in the is_member_of claim in the id token
-rokwire_event_manager_group = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire events manager'
-rokwire_events_uploader = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire ems events uploader'
-rokwire_events_web_app = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire events web app'
-rokwire_events_approvers = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire event approvers'
-rokwire_group_admins = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire groups access'
-rokwire_app_config_manager_group = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire app config manager'
+# rokwire_event_manager_group = 'urn:mace:' + org + \
+#     ':authman:app-rokwire-service-policy-rokwire events manager'
+# rokwire_events_uploader = 'urn:mace:' + org + \
+#     ':authman:app-rokwire-service-policy-rokwire ems events uploader'
+# rokwire_events_web_app = 'urn:mace:' + org + \
+#     ':authman:app-rokwire-service-policy-rokwire events web app'
+# rokwire_app_config_manager_group = 'urn:mace:' + org + \
+#     ':authman:app-rokwire-service-policy-rokwire app config manager'
+
+rokwire_event_manager_group = 'events_manager'
+rokwire_events_uploader = 'events_uploader'
+rokwire_events_web_app = 'events_web_app'
+rokwire_app_config_manager_group = 'app_config_manager'
 
 ROKWIRE_EVENT_WRITE_GROUPS = [rokwire_event_manager_group, rokwire_events_uploader, rokwire_events_web_app,
```

Comment:
`rokwire_event_approvers` and `rokwire_group_admins` variables are not initialzed. These are the corresponding group names: 

```
rokwire_events_approvers = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire event approvers'	
rokwire_group_admins = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire groups access'
```